modules/ossm-rn-new-features.adoc

```diff
@@ -37,6 +37,10 @@ Result – If changed, describe the current user experience
 |===
 
 
+== New features {ProductName} 2.0.3
+
+This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
```

Comment:
In addition, this release has the following improvements:

* Added an option to must-gather to allow gathering only specific control plane namespace (OSSM-351)
* Improved performance for control planes with hundreds of namespaces